tests/common.py

```diff
@@ -1111,7 +1111,7 @@ def _build_and_run(self, filename, expected_output, args=[], output_nicerizer=No
               self.assertContained(o, js_output)
           else:
             self.assertContained(expected_output, js_output)
-            if check_for_error:
+            if assert_returncode == 0 and check_for_error:
```

Comment:
Is this related?

Reply:
Yes.. because some of the tests being converted are ones that test sanitizers and therefore have "ERROR" in their expected output (but they also set assert_returncode=NON_ZERO).